pkg/api/osa/vlabs/convert.go

```diff
@@ -0,0 +1,55 @@
+package vlabs
```

Comment:
Also, need the reverse convertor from internal to public model. A TODO for the future.